alot/settings/manager.py

```diff
@@ -504,7 +504,7 @@ def get_main_addresses(self):
         return [a.address for a in self._accounts]
 
     def get_addresses(self):
-        """returns addresses of known accounts including all their aliases"""
+        """returns addresses and aliases of known accounts, excluding regex aliases"""
```

Comment:
This is used only for tab-completing the "from" headers in compose commands right?